data-science topic added (#1964)
github/explore: topics/data-science/index.md

---
aliases: datasciences, data-science-project, data-science-algorithm
display_name: Data Science
short_description: Data science is an interdisciplinary field focused on extracting knowledge from typically large data sets.
topic: data-science
related: data-analysis, data-mining, machine-learning, big-data, data-visualization
wikipedia_url: https://en.wikipedia.org/wiki/Data_science
---

> Data science is an inter-disciplinary field that uses scientific methods, processes, algorithms, and systems to extract knowledge and insights from many structural and unstructured data. The field encompasses analysis, preparing data for analysis, and presenting findings to inform high-level decisions in an organization. 

Reviewer: Could we restructure this to be a little easier to read:
```suggestion
Data science is an inter-disciplinary field that uses scientific methods, processes, algorithms, and systems to extract knowledge from structured and unstructured data. Data scientists perform data analysis and preparation, and their findings inform high-level decisions in many organizations.
```

Author: I think rewording makes it easier to read for sure. It was very technical before. This is nice 😁
